common/metrics.py

```
@@ -0,0 +1,223 @@
+
+        Usage:
+            with metrics.time_cache_operation('Insult', 'load'):
+                # Your cache operation here
+                pass
+        """
+        return cls.cache_operation_duration.labels(
+            model=model, operation=operation
+        ).time()
+
+    @classmethod
+    def time_database_query(cls, model: str, status: str = "success"):
```

Ruff (0.12.2)

Missing return type annotation for classmethod `time_database_query`

(ANN206)